news/_posts/2020-05-11-cms-security-summit.md

```diff
@@ -0,0 +1,95 @@
+
+After spending the first evening with stakeholders and meeting the newcomers over a few beers, we spent two days at Google Munich office and [Information Security Hub (ISH)](https://www.ish-muc.com/) to speak about what we did since the last years and talk about what's next.
+And of course learnt about each others.
+
+
+We had ligthingtalks as well as conferences about different technologies around the security and data privacy.
+
+As the last year, components like [Strict Content Security Policy](https://bit.ly/strict-csp), [Trusted Types](https://bit.ly/tt-spec) and [Scripting Policy](https://bit.ly/scripting-policy) are again discussed and brought to the forefront.
+
+[Cross-Origin Resource Policy (CORP)](https://fetch.spec.whatwg.org/#cross-origin-resource-policy-header), [Cross-Origin-Embedded-Policy](https://bit.ly/coep-spec) and [Securer Contexts](https://bit.ly/securer-contexts) to enhance isolation and mitigate side-channel attacks.
+
+We also discover the interesting [OpenSK](https://github.com/google/OpenSK) as a [Tock OS](https://www.tockos.org/) application, an open-source security key to be used for multi-factor authentication.
+
+We talked again about the potential of PSR-9 and PSR-10 PHP Standards Recommendations, but since the last year, nothing really moved which is a little sad.
+
+[Snyk](https://snyk.io) gave us a talk about [Javascript Prototype Pollution](https://snyk.io/vuln/SNYK-JS-LODASH-450202) which reminds us that we must pay attention about dependencies we used, even if they are famous.
+It's mandatory to update them as soon as possible even this results in [BC Breaks](bc-breaks).
+
+And to finish and interresting talk about [Virus Total API](https://developers.virustotal.com/reference) to cluster and classify potential malware on the web.
+
+
+# Breakout Sessions
+
+The rest of the time was used for breakout sessions, allowing participants to share ideas, experiences around many differents themes defined before the summit.
+
+## Rapid Detection and prevention
+
+Content Management Systems are used by the majority of websites in the world and the number of attacks are still increased every years.
+This has been identified as a priority for everyone to be able to quickly detect attacks and act as fast as possible accordingly.
+
+On the PrestaShop side, the open source project decided to greatly improve the communication around security issues and get closer to (YesWeHack)[https://yeswehack.com/] for the implementation of a Bug Bounty program.
+By doing so, we will detect security issues easier, faster and provide a security fix as fast as possible regarding the criticiy of the issue.
+
+
+## Automatic updates
+
+As everyone already know, the principal risk for a software is its age, keeping it updated is mandatory and considerably reduce the risk of having security issues.
+Even if you think at a T time there are no problems, nobody and nothing can not predict the future, a 0 day on PHP, Apache, Nginx ...
```

Comment:
```suggestion
Even if you think at a T time there are no problems, nobody and nothing can predict the future, a 0 day on PHP, Apache, Nginx ...
```

This sentence looks strange, even with the correction

Reply:
I commit your suggestion, but you're right it's a little bit weird :sweat: